TCMSApp/src/client/app/runs/runs-exec/runs-execute.controller.js

```diff
@@ -0,0 +1,49 @@
+(function (){
+    'use strict';
+
+    angular
+        .module('app.runsExec')
+        .controller('RunsExecuteController', RunsExecuteController);
+
+   RunsExecuteController.$inject = ['$stateParams', '$state','logger','moment' ];
+
+   function RunsExecuteController($stateParams, $state) {
+
+       var vm = this;
+       vm.run = JSON.parse(JSON.stringify($stateParams.run));//creating of the deep copy
+       vm.stateName = $state.current.name;
+       vm.progress = getProgress();
+       vm.suites = getSuites();
+       vm.selectedSuite = vm.suites[0];
+       vm.selectedTest = vm.run.tests[0];
+       vm.styles = {
+           backColorEdit: '#fff',
```

Comment:
styles here too?
have you considered having the switch between run and edit outside the ui-view with run and edit?
to be discussed